modules/update_helper_checklist/update_helper_checklist.module

```diff
@@ -62,10 +63,10 @@ function _update_helper_checklist_checklistapi_checklist_items() {
           $entry = Update::load($update_key);
 
           $status = ($entry && $entry->wasSuccessfulByHook()) ? TRUE : FALSE;
-          if ($status && !empty($update['#description_successful'])) {
+          if ($entry && $status && !empty($update['#description_successful'])) {
```

Comment:
I'm not sure if we need to check for `$entry` in combination with `$status`, because it's already checked before with this statement: `$status = ($entry && $entry->wasSuccessfulByHook()) ? TRUE : FALSE;`